app/sidekiq/lighthouse/submit_benefits_intake_claim.rb

```diff
@@ -51,14 +50,13 @@ def perform(saved_claim_id)
       response = @lighthouse_service.upload_doc(**payload)
 
       if response.success?
-        log_message_to_sentry('CentralMail::SubmitSavedClaimJob succeeded', :info, generate_sentry_details)
+        Rails.logger.info('Lighthouse::SubmitBenefitsIntakeClaim succeeded', generate_sentry_details)
```

Comment:
Consider renaming `generate_sentry_details` to `generate_log_details` to be more clear where the information is going